### PR TITLE
feat:created wishManagement page

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

File: wishManagement/wish.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+}
+
+#card-container{
+    background-color: transparent;
+}
+
+.wish-add{
+    background-color: var(--light-pink) !important;
+    color:var(--violet) !important;
+    border:none !important;
```
Comment (vinc3nati) [inline wishManagement/wish.css]: Suggestion: don't use !important wherever possible

**Review by vinc3nati — APPROVED** [review]

**Review by rahulbakshi4 — APPROVED** [review]
Looks good!!!
Just a suggestion try to add some hover states on the buttons on the card.

File: wishManagement/wish.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                      <h5 class="card-subtitle">Blue Jacket</h5>
+                      <h3 class="card-title">₹150</h3>
+                      <div class="card-actions flex-container flex-columns">
+                      <a href="#" class="card-btn-1 mb">Add to Cart</a>
+                      <a href="#" class="card-btn-2 wish-add">Remove from Wishlist</a>
+                    </div>
+                    </div>
+                  </div>
+                  <div class="card card-basic card-shadow">
+                    <img src="https://picsum.photos/id/1005/340/196" class="card-img-top" alt="...">
```
Comment (lomesshh) [inline wishManagement/wish.html]: give proper value to alt attribute

**Review by lomesshh — APPROVED** [review]
just try to avoid use of "! important"
else everything else looks good

File: wishManagement/wish.css (hunk 1 above)
Comment (KamiyaGaikwad) [reply wishManagement/wish.css]: Thank you for pointing it out, removed it

File: wishManagement/wish.html (hunk 2 above)
Comment (KamiyaGaikwad) [reply wishManagement/wish.html]: Thanks for pointing it out, done

**Comment by KamiyaGaikwad** [comment]
Thank you for your suggestion @rahulbakshi4. i added hover on buttons

**Comment by KamiyaGaikwad** [comment]
Thank you for pointing it out @lomesshh, I have removed it

**Pull request merged**